g2p/mappings/langs/crj/config.yaml

```diff
@@ -3,7 +3,7 @@
 mappings:
   - display_name: Northern East Cree Normalization
```

Comment:
I think I would change the display name for each of these to "lang_name equivalency" or "lang_name character equivalency". You probably want to get rid of the word Normalization in the display names too.

I remember we agreed on the short form "equiv", did we agree on a long form?